### PR TITLE
Fix UI test names according to new naming conventions (part 3)

### PR DESCRIPTION
**Pull request by abalakh**
Also updated some negative API tests

### CODE REVIEW

**Comment by abalakh** [comment]
```python
py.test -v tests/foreman/ui/test_lifecycleenvironment.py 
================================ test session starts =================================
platform linux2 -- Python 2.7.10, pytest-2.8.2, py-1.4.30, pluggy-0.3.1 -- /home/andrii/env/bin/python
cachedir: .cache
rootdir: /home/andrii/workspace/robottelo, inifile: 
collected 4 items 

tests/foreman/ui/test_lifecycleenvironment.py::LifeCycleEnvironmentTestCase::test_positive_create <- robottelo/decorators.py PASSED
tests/foreman/ui/test_lifecycleenvironment.py::LifeCycleEnvironmentTestCase::test_positive_create_chain <- robottelo/decorators.py PASSED
tests/foreman/ui/test_lifecycleenvironment.py::LifeCycleEnvironmentTestCase::test_positive_delete <- robottelo/decorators.py PASSED
tests/foreman/ui/test_lifecycleenvironment.py::LifeCycleEnvironmentTestCase::test_positive_update <- robottelo/decorators.py PASSED

============================= 4 passed in 200.15 seconds =============================
```

**Comment by abalakh** [comment]
Pending @oshtaier's review :)

File: tests/foreman/ui/test_activationkey.py [hunk 1]
```diff
@@ -764,7 +774,8 @@ def test_negative_update_activation_key_with_invalid_limits(self):
                     )
 
     @run_only_on('sat')
-    def test_usage_limit(self):
+    @tier3
+    def test_negative_usage_limit(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_activationkey.py]: `@skip_if_not_set('clients')`

File: tests/foreman/ui/test_adusergroup.py [hunk 2]
```diff
@@ -57,9 +62,10 @@ def tearDown(self):
             session.nav.go_to_users()
             if self.user.search(self.ldap_user_name):
                 self.user.delete(self.ldap_user_name)
-        super(ADUserGroups, self).tearDown()
+        super(ActiveDirectoryUserGroupTestCase, self).tearDown()
 
-    def test_adusergroup_admin_role(self):
+    @tier3
+    def test_positive_add_admin_role(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_adusergroup.py]: `@skip_if_not_set('ldap')`

File: tests/foreman/ui/test_adusergroup.py [hunk 3]
```diff
@@ -89,12 +95,13 @@ def test_adusergroup_admin_role(self):
...
             self.ldap_user_passwd,
-        ) as session:
+        ):
             self.assertIsNotNone(self.login.wait_until_element(
                 (strategy, value % self.ldap_user_name)
             ))
 
-    def test_adusergroup_foreman_role(self):
+    @tier3
+    def test_positive_add_foreman_role(self):
```
Comment (oshtaier) [inline tests/foreman/ui/test_adusergroup.py]: `@skip_if_not_set('ldap')`

File: tests/foreman/ui/test_adusergroup.py [hunk 4]
```diff
@@ -57,9 +62,10 @@ def tearDown(self):
             session.nav.go_to_users()
             if self.user.search(self.ldap_user_name):
                 self.user.delete(self.ldap_user_name)
-        super(ADUserGroups, self).tearDown()
+        super(ActiveDirectoryUserGroupTestCase, self).tearDown()
 
-    def test_adusergroup_admin_role(self):
+    @tier3
```
Comment (oshtaier) [inline tests/foreman/ui/test_adusergroup.py]: Don't think that is tear3

**Comment by oshtaier** [comment]
ACK pending comments

File: tests/foreman/ui/test_activationkey.py (hunk 1 above)
Comment (abalakh) [inline tests/foreman/ui/test_activationkey.py]: :+1:

**Comment by oshtaier** [comment]
ACK

**Pull request merged**